optuna/storages/_rdb/storage.py

```diff
@@ -608,14 +608,19 @@ def _update_trial(
                 trial_model.datetime_complete = datetime_complete
 
             if value is not None:
-                trial_model.value = value
+                trial_values = models.TrialValueModel.where_trial_id(trial_id, session)
+                trial_values_dict = {v.objective: v for v in trial_values}
+                for objective, v in enumerate([value]):
+                    if objective in trial_values_dict:
+                        trial_values_dict[objective].value = v
+                        session.add(trial_values_dict[objective])
+                    else:
+                        trial_model.values.extend(
+                            [models.TrialValueModel(objective=objective, value=v)]
```

Comment:
Oh, the trial_id is automatically added?

Reply:
I think so, but it's good for the code readability to specify the `trial_id`. I will add it.

Reply:
I think so. I implemented this part in reference to the other `Trial***Model`. For example, the extension of the ｀`TrialUserAttributeModel` in the same function. Should I add the `trial_id` for code readability?

Reply:
It's a details but yes that'd be clearer. There by the way were other similar occurrences where we explicitly specified the id argument.